source/server/config_validation/server.h

```diff
@@ -174,7 +174,12 @@ class ValidationInstance final : Logger::Loggable<Logger::Id::main>,
   void initialize(const Options& options,
                   const Network::Address::InstanceConstSharedPtr& local_address,
                   ComponentFactory& component_factory);
-
+  const Options& options_;
+  ProtobufMessage::ProdValidationContextImpl validation_context_;
+  Stats::IsolatedStoreImpl& stats_store_;
+  Api::ApiPtr api_;
+  Event::DispatcherPtr dispatcher_;
+  ThreadLocal::InstanceImpl thread_local_;
```

Comment:
What's the reasoning behind this change?

Reply:
I cannot recall. shutdown should have resolve the issue. Reverted